src/storage/mod.rs

```diff
@@ -43,6 +45,7 @@ pub use self::engine::raftkv::RaftKv;
 pub use self::engine::{new_local_engine, CFStatistics, Cursor, Engine, Error as EngineError,
                        FlowStatistics, Iterator, Modify, ScanMode, Snapshot, Statistics,
                        StatisticsSummary, TEMP_DIR};
+pub use self::gc_worker::GC_MAX_PENDING;
```

Comment:
I think you can simply make gc_worker public so that we don't need to import this constant here, just like other sub modules. Also this is more friendly if there are more constants in future.